packages/calypso-products/src/plans-list.tsx

```diff
@@ -1588,7 +1588,16 @@ const getPlanJetpackSecurityT1Details = (): IncompleteJetpackPlan => ( {
 		),
 	getFeaturedDescription: () =>
 		translate(
-			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection.'
+			'This bundle includes:{{ul}}{{li}}VaultPress Backup{{/li}}{{li}}Scan{{/li}}{{li}}Akismet Anti-spam{{/li}}{{/ul}}',
+			{
+				components: {
+					ul: <ul />,
+					li: <li />,
+					br: <br />,
```

Comment:
It looks like <br /> is not being used in this string, can probably remove here and in the comment

Reply:
Thanks, I fixed it 👍